Quickstart/azure-search-quickstart/azure-search-quickstart/Address.Methods.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace azure_search_quickstart
+{
+    using System;
+    using System.Text;
+    using Newtonsoft.Json;
+
+    public partial class Address
+    {
+        // This implementation of ToString() is only for the purposes of the sample console application.
+        // You can override ToString() in your own model class if you want, but you don't need to in order
+        // to use the Azure Search .NET SDK.
+
+        public override string ToString()
+        {
+            var builder = new StringBuilder();
+
+            if (!IsEmpty)
+            {
+                builder.AppendFormat("{0}\n{1}, {2} {3}\n{4}", StreetAddress, City, StateProvince, PostalCode, Country);
+            }
+
+            return builder.ToString();
+        }
+
+        [JsonIgnore]
+        public bool IsEmpty => String.IsNullOrEmpty(StreetAddress) &&
```

Comment:
This doesn't need to be public. Please make it `private`.

Reply:
Done

Reply:
Ok, I didn't see the usage in `Hotel.ToString()`.